core/src/apps/bitcoin/keychain.py

```diff
@@ -150,6 +150,8 @@ def validate_path_against_script_type(
             append(PATTERN_GREENADDRESS_B)
         if coin.coin_name in BITCOIN_NAMES:
```

Comment:
If these paths are only used for multisig, which they are in the already existing code, then the condition to add them should be:
```py
if coin.coin_name in BITCOIN_NAMES and multisig:
    [...]
```

Reply:
change in 62caf4c12f6eb822fbe7b24422c95b972a483301